src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -23,6 +221,167 @@ g.test('depth_write_disabled')
   .desc(`Tests render results with depth write disabled`)
   .unimplemented();
 
+g.test('stencil_compare_func')
+  .desc('Tests that stencil comparison functions works as expected.')
+  .params(u =>
+    u //
+      .combineWithParams([
+        {
+          stencilCompare: 'always',
+          stencilColor: lessCompareColor,
```

Comment:
I think we only need to see whether the fragment passed or failed the stencil test, so we only need 2 colors. Why are different `stencilColor`s needed for different test cases?

Reply:
I wrote this test referring to the end2end dawn reference test. So I needed to use the different `stencilColor` to cover the test. Do you think it's better to only use 2 colors?

[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/tests/end2end/DepthStencilStateTests.cpp;l=505?q=depthStencilState&ss=chromium